locate.go

```diff
@@ -0,0 +1,82 @@
+package meilisearch
+
+// Locate is localized attributes for index in search
+//
+// https://www.meilisearch.com/docs/reference/api/settings#localized-attributes-object
+type Locate string
+
+const (
```

Comment:
Can we avoid to have this list? I would rather we accept a string so that if we remove/add new languages (as we did in v1.11) we don't have to update this, please?

I would like to keep the structure simple for this and avoid additional work when Meilisearch is adding new features 🙏

Reply:
Sure